packages/api/src/express/chatRouter.ts

```diff
@@ -0,0 +1,75 @@
+import express from 'express';
+import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
+import {
+  experimental_createMCPClient as createMCPClient,
+  InvalidToolArgumentsError,
+  NoSuchToolError,
+  ToolExecutionError,
+  streamText,
+} from 'ai';
+import { anthropic } from '@ai-sdk/anthropic';
+import { defaultLogger } from '@unchainedshop/logger';
+
+const chatRouter = express.Router();
+const { CHAT_API_PATH = '/chat', ANTHROPIC_API_KEY, ROOT_URL = 'http://localhost:4010' } = process.env;
+
+const errorHandler = (error: any): string => {
+  if (NoSuchToolError.isInstance(error)) return 'NoSuchToolError';
+  if (InvalidToolArgumentsError.isInstance(error)) return 'InvalidToolArgumentsError';
+  if (ToolExecutionError.isInstance(error)) return 'ToolExecutionError';
+  if (error?.message?.toLowerCase() === 'forbidden') return 'NetworkError';
+  defaultLogger.error(error);
+  return `Failed to stream response: ${error?.message || 'Unknown error'}`;
+};
+
+chatRouter.post(CHAT_API_PATH, async (req: any, res) => {
+  const { messages } = req.body;
+  if (!ANTHROPIC_API_KEY) {
```

Comment:
That can't be: If i search for "anthropic" in the code base, no single usage should appear at all under packages. ANY MODEL has to be supported.